### ISSUE TITLE
`FAIL: tests/multiword-is-t.exp` on 0.7.5

### ISSUE THREAD

**Issue by polyzen** [jwilk/mwic#7]
```
======================================================================                                                                  
FAIL: tests/multiword-is-t.exp                                                                                                          
----------------------------------------------------------------------                                                                  
Traceback (most recent call last):                                                                                                      
  File "/home/user/pkg/build/mwic/src/mwic-0.7.5/tests/../tests/test_blackbox.py", line 116, in _test                                 
    _test_text(self.path)                                                                                                               
  File "/home/user/pkg/build/mwic/src/mwic-0.7.5/tests/../tests/test_blackbox.py", line 80, in _test_text                             
    assert_multi_line_equal(expected, text)                                                                                             
AssertionError: 'could’t:\n| It could’t ...\n     ^^^^^^^\[433 chars]\n\n' != 'ai’t:\n| It ai’t ...\n     ^^^^\n\ncould’[437 chars]\n\n'
```

**Comment by jwilk** [comment]
Does 5591924178e66d900cccd3e30a3fb3b4e3dfd83e fix it for you?
(Apparently [ai](https://www.dictionary.com/browse/ai) is a three-toed sloth…)

**Comment by polyzen** [comment]
It does, thank you.

**Comment by jwilk** [comment]
Fixed in 0.7.6.
